web/packages/teleterm/src/mainProcess/mainProcessClient.ts

```diff
@@ -199,5 +202,40 @@ export default function createMainProcessClient(): MainProcessClient {
         args
       );
     },
+    checkForAppUpdates() {
+      return ipcRenderer.invoke(MainProcessIpc.CheckForAppUpdates);
+    },
+    downloadAppUpdate() {
+      return ipcRenderer.invoke(MainProcessIpc.DownloadAppUpdate);
+    },
+    cancelAppUpdateDownload() {
+      return ipcRenderer.invoke(MainProcessIpc.CancelAppUpdateDownload);
+    },
+    quitAndInstallAppUpdate() {
+      return ipcRenderer.invoke(MainProcessIpc.QuiteAndInstallAppUpdate);
+    },
+    changeAppUpdatesManagingCluster(clusterUri) {
```

Comment:
Is it a problem if a rogue renderer changes the managing cluster to a cluster that is not in the app?

It doesn't feel like it should be a problem because `getClusterVersions` always works on data returned from tshd. So the worst thing that this IPC can cause is simply forcing the user to pick a managing cluster again.

Reply:
Correct, `resolveAutoUpdatesStatus` completely ignores clusters that do not exist. 
So if you have a single cluster and `changeAppUpdatesManagingCluster` sets a different cluster, you won't notice anything. 
If you had a cluster selected, you will need to select it again.